### PR TITLE
Publish Extension Wallet as Zip File

### PR DESCRIPTION
**Pull request by h0ngcha0**

### CODE REVIEW

**Comment by h0ngcha0** [comment]
@polarker I have renmed `get-alephium` to `get-extension-wallet`, do we need to create a `get-extension-wallet` package on our NPM registry?

**Comment by polarker** [comment]
> do we need to create a get-extension-wallet package on our NPM registry?

Yes. `npm publish --access public` could create the package if it does not exist.

File: .eslintignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 scripts
-packages/get-alephium
\ No newline at end of file
+packages/get-extension-wallet
```
Comment (polarker) [inline .eslintignore:2]: Is this on purpose to ignore the lint for the package?

**Comment by polarker** [comment]
Two feedbacks:
1. it seems a few auto-generated files are included, are they needed?
2. is it better to remove the zip file from the repo? git clone would be very slow with zip file histories.

**Comment by h0ngcha0** [comment]
> Two feedbacks:
> 
> 1. it seems a few auto-generated files are included, are they needed?
> 2. is it better to remove the zip file from the repo? git clone would be very slow with zip file histories.

Oops, great catch, zip file is not intended to be checked in.

File: .eslintignore (hunk 1 above)
Comment (h0ngcha0) [reply .eslintignore]: Great catch, this was used during development, now it can be removed :+1:

**Comment by h0ngcha0** [comment]
@polarker Thanks for the review <3  I have fixed the comments. (Did a force push to remove the zip file that was checked in unintentionally)

**Review by polarker — APPROVED** [review]
LGTM 👍

**Pull request merged**